qt/aqt/mediasrv.py

```diff
@@ -599,6 +602,210 @@ def handle_on_main() -> None:
     return b""
 
 
+def editor_update_note() -> bytes:
```

Comment:
mediasrv is a bit of a mess now with all this screen-specific logic. Will look into a cleaner solution, maybe by allowing screens to register their own handlers.

Reply:
AnkiMobile currently does this - each screen registers only the endpoints that are needed, so other endpoints give an error. As we get further along in our porting and we start transitioning from one 'page' to another in the same webview session, it'll probably become less practical to do that, though.